scripts/index.js

```diff
@@ -56,10 +60,10 @@ function getMovieDetails(moviesArr) {
     })
     Promise.all(requests)
         .then(responses => {
-            movieResults = responses.filter((movie) => {
+            const movieResults = responses.filter((movie) => {
                 return movie.Title !== "N/A" && movie.Poster !== "N/A"
             })
-            renderMovieResults()
+            renderMovieResults(movieResults)
```

Comment:
We create `movieResults` here based on the API response, and pass the list of movies down into the `renderMovieResults` function